src/main.c

```diff
@@ -2408,11 +2411,17 @@ void handle_mouse_input() {
     }
 }
 
-void handle_movement(double dt) {
+
+void handle_movement(double dt, bool *is_auto_walking_ptr, bool *allow_next_autowalk_press_ptr, bool *do_auto_walk_ptr) {
```

Comment:
I would recommend changing the three parameters you added into a struct containing those variables to make the code cleaner.

Reply:
I would do this, but compared to some other changes in master by other members, most use a similar variable set-up like mine.